<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="configuration" />
    <arg name="output" />

    <node pkg="explore_lite" type="explore" name="explorer" respawn="true" output="$(arg output)" >
	    <rosparam file="$(arg configuration)" />
    </node>

</launch>
